internal/pkg/term/prompt/prompt.go

```diff
@@ -117,8 +162,8 @@ func stdio() survey.AskOpt {
 
 func icons() survey.AskOpt {
 	return survey.WithIcons(func(icons *survey.IconSet) {
-		icons.Question.Format = "cyan"
-		icons.Help.Format = "white"
+		icons.Question.Format = "cyan+b"
+		icons.Help.Format = "default+ih:default"
```

Comment:
Can we document what impacts this formatting has?

Reply:
Good idea! Added comments